src/test/java/tests/StakeKeyTests.java

```diff
@@ -0,0 +1,31 @@
+package tests;
+
+import base.BaseTest;
+import microservices.stakeKey.steps.StakeKeySteps;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class StakeKeyTests extends BaseTest {
+    private StakeKeySteps stakeKeySteps = new StakeKeySteps();
+    @Test(description = "get a stake detail by address", groups = {"stakeKey"})
+    public void getStakeByAddress(){
+        stakeKeySteps.getStakeByAddress("addr_test1qr53akzyd4949txn5hu583yu0xatcvp2efec9tm56jpeg6xkfjf77qy57hqhnefcqyy7hmhsygj9j38rj984hn9r57fsq48dyr")
```

Comment:
You should declare an variable like String address = "addr_test1qr53akzyd4949txn5hu583yu0xatcvp2efec9tm56jpeg6xkfjf77qy57hqhnefcqyy7hmhsygj9j38rj984hn9r57fsq48dyr" and put above fucntion test cause in the future many we have 2 enviroment hence we have 2 different data input

Reply:
okay, i'll do that